src/client/MetaSession.cc

```diff
@@ -31,6 +32,14 @@ void MetaSession::dump(Formatter *f) const
   f->dump_stream("last_cap_renew_request") << last_cap_renew_request;
   f->dump_unsigned("cap_renew_seq", cap_renew_seq);
   f->dump_int("num_caps", caps.size());
+  f->open_array_section("caps");
+  for (xlist<Cap*>::const_iterator p = caps.begin(); !p.end(); ++p) {
```

Comment:
This will need to be an option to `session ls`. The quantity of output will be too much for production clusters.

I like the idea though!

Reply:
@batrick How about this?

Reply:
I was thinking a flag like

`ceph tell mds.foo session ls --cap-dump`

Reply:
> I was thinking a flag like
> 
> `ceph tell mds.foo session ls --cap-dump`

done.